apps/laboratory/tests/shared/constants/devices.ts

```diff
@@ -1 +1 @@
-export const DEVICES = ['Desktop Chrome', 'Desktop Firefox', 'Desktop Safari']
+export const DEVICES = ['Desktop Chrome', 'Desktop Brave', 'Desktop Firefox']
```

Comment:
Why did we remove Safari?

Reply:
It was disabled for some time in the utility function, so disabled it here as well. Its not working on ubuntu runners. Will debug separately.